corehq/apps/data_interfaces/static/data_interfaces/js/archive_forms.js

```diff
@@ -0,0 +1,78 @@
+hqDefine("data_interfaces/js/archive_forms", function() {
+    var managementSelector = '#data-interfaces-archive-forms',
+        allFormsButtonSelector = managementSelector + ' input[name="select_all"]',
+        checkboxesSelector = managementSelector + ' input.xform-checkbox',
+        indicatorSelector = '#count_indicator';
+
+    function updateFormCounts() {
+        var selectedCount = $(managementSelector + ' input.xform-checkbox:checked').length;
+        $(".selectedCount").text(selectedCount);
+        enableDisableButton(selectedCount);
+    }
+
+    function enableDisableButton(count){
```

Comment:
seems like `toggleButton` might have been a better name for this, but I see it was already like this

Reply:
yeah, i should have done that as long as i was fixing the casing. updated.